core/controllers/base_test.py

```diff
@@ -1984,3 +1984,55 @@ def test_image_upload_and_download(self):
             )
             filename = response_dict['filename']
         self.logout()
+
+
+class UrlPathNormalizationTest(test_utils.GenericTestBase):
+    """Tests that ensure url path arguments are normalized"""
+
+    class MockHandler(base.BaseHandler):
+        URL_PATH_ARGS_SCHEMAS = {
+            'mock_list': {
+                'schema': {
+                    'type': 'custom',
+                    'obj_type': 'JsonEncodedInString'
+                }
+            },
+            'mock_int': {
+                'schema': {
+                    'type': 'int'
+                }
+            }
+        }
+        HANDLER_ARGS_SCHEMAS = {
+            'GET': {}
+        }
+
+        def get(self, mock_list, mock_int):
+            if not isinstance(mock_list, list):
+                raise self.InvalidInputException(
+                        'Expected arg mock_list to be a list. Was type %s' %
```

Comment:
Not sure why the linter's not catching this, but it should be (here and below):

```
                raise self.InvalidInputException(
                    'Expected arg mock_list to be a list. Was type %s' %
                    type(mock_list))
```

due to 4-space indentation.

/cc @Hudda @DubeySandeep @AdityaDubey0

Reply:
Done

Reply:
Er -- not quite done, this should be indented by 4, not 8. (Similarly below.)

Reply:
Actually done